### ISSUE TITLE
State of Repo

### ISSUE THREAD

**Issue by wadewilliams** [Tivix/django-rest-auth#568]
Just curious how maintained this repo is. No `CONTRIBUTORS.md` and a lot of open issues. We're happy to help out if help's needed.

**Comment by Raisins** [comment]
The repo is on pause and is currently unsupported.

**Comment by auvipy** [comment]
I am going to vendor this packge on one of my package named drf-extensions

**Comment by iMerica** [comment]
"On pause" doesn't make sense. Transfer the repo to someone willing to maintain it or ask for maintainers in a pinned Github issue.

**Comment by auvipy** [comment]
> "On pause" doesn't make sense. Transfer the repo to someone willing to maintain it or ask for maintainers in a pinned Github issue.

I agree with you. I am planning to create a better version of this package as my recent projects need them.

**Comment by lggwettmann** [comment]
Hey guys,

Could you please post this on the ReadMe asap? I installed this in a new project only to find out after I'm almost completely done, that I run into bugs and this repo is unmaintained - and it didn't look like it at first sight.

Let's prevent others from making the same mistake. Thanks.

**Comment by cobyrne09** [comment]
We need to get ahold of Tivix... they have a support contact. I'll give it a try now.

**Comment by iMerica** [comment]
I'm willing to maintain this project under a few conditions:
- Transfer this to a new Github group called django-rest-auth or alternatively transfer the repo to me and I'll create it. 
-  Remove everyone who has commit access, but is not maintaining the repo. This would make everyone sleep better at night.

Otherwise, I can just publish a fork under a new name (e.g `django-rest-auth-redux`)

**Comment by cobyrne09** [comment]
@iMerica I think it's best we just start over with a new name. I have little faith that Tivix will implement those steps, considering these discussions have been been going on for six months and we haven't heard a word from them.

**Comment by cobyrne09** [comment]
I received this response from Tivix: 

`Hi,

I understand the issue. We don't have capacity to support it internally sadly.

Rather than transfer the maintenance to someone else, our preference would be if you and few other like minded people would like to fork and build / push to pypi something like "django-rest-auth-improved" or something like that, we will support it and mark our repo as deprecated and point all users and issue posters to your repo / pypi package.

Thoughts?`

It definitely helps to have their buy-in and will help to have an official message directing users to the new repo. So wether through @iMerica's repo or someone else's, we should proceed with this.

**Comment by iMerica** [comment]
Okay that sounds like a "no" to me.  I'll just fork it - it's not really important if they "buy-in" or not. This is open source – users vote with their feet.

**Comment by iMerica** [comment]
[Dj REST-AUTH](https://github.com/jazzband/dj-rest-auth) is now live. I'm using it in production on a small project and everything is running smoothly so far. 

The public API is identical, just a different namespace (`dj_rest_auth`) to mitigate any confusion.

All bugs and feedback is welcome in the Github issues of that repo.
- [Docs](https://dj-rest-auth.readthedocs.io/en/latest/)
- [Repo](https://github.com/jazzband/dj-rest-auth)
- [CI (Circle CI)](https://app.circleci.com/github/jazzband/dj-rest-auth/pipelines)
- [PyPi Package](https://pypi.org/project/dj-rest-auth/)

**Comment by shawngustaw** [comment]
@iMerica can you enable issues on the forked repo? If we can get enough eyes on it we should reach out to Tom to point the docs at your fork https://github.com/encode/django-rest-framework/issues/7147

**Comment by iMerica** [comment]
Thanks for bringing this to my attention. Trying to enable it now.

**Comment by mohmyo** [comment]
Please, update the repo readme with current status and mention the newer fork [dj-rest-auth](https://github.com/jazzband/dj-rest-auth),
Thanks.

**Comment by BarnabasSzabolcs** [comment]
OK, I cross-referenced all the PR's to the new repo, hopefully lots of valuable contribution will not get lost and will get merged to the maintained codebase. (Some of those PR's I'd also need a lot.)

Best of luck,
Barney

**Comment by mohmyo** [comment]
We are about a year-long and the readme is not updated till now! unbelievable!
Stop the confusion for new ppl coming in and add a clear statement about the repo on the readme file

**Comment by weslord** [comment]
It looks like a deprecation message with a link to the the actively maintained fork (https://github.com/iMerica/dj-rest-auth) was added to the wrong repo by mistake:
- https://github.com/Tivix/django-cron/commit/ed558446c4daf0119883d4cce436815bb4a10960
- https://github.com/Tivix/django-cron/issues/177

I've duplicated that commit in a PR here: #657
